### ISSUE TITLE
Make the Theme Wizard generate MV3 static themes 

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons#1912]
As discussed with diox , I'm opening an issue:

The theme wizard is generating MV2 versions for now 

MV3 is coming starting January and submission, installation and updates were tested in [QA-1668](https://mozilla-hub.atlassian.net/browse/QA-1668)

At one point in the future, the theme wizard should generate MV3 themes too.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDSRV-184)

**Comment by diox** [comment]
Let's confirm that it's ok to do with web-ext folks but otherwise should be ok. We don't need to care about the existing themes yet.

**Comment by willdurand** [comment]
> Let's confirm that it's ok to do with web-ext folks but otherwise should be ok. We don't need to care about the existing themes yet.

Nope, thi is not OK yet... https://bugzilla.mozilla.org/show_bug.cgi?id=1917613

**Comment by diox** [comment]
:+1: we are not in a hurry to do it anyway. I've marked this as blocked, we'll want to wait for this to be fixed (and any other blockers we might discover) and probably to be rolled out in the next ESR as well...

**Comment by willdurand** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1917613 has been fixed in Firefox 132. While it was less concerning than initially thought, we'd still recommend to use `132` as the default min version for MV3 themes.

FWIW I've uploaded a [MV3 theme on -dev](https://addons-dev.allizom.org/en-US/firefox/addon/reference-static-theme-mv3/) and everything is working flawlessly: installs, updates, etc. But we don't know what has changed between 109 (first version with MV3 support) and 132, so while themes work fine in both Release and Nightly right now, it's possible that some Firefox versions break in a more obvious (and bad) way.

Note also that MV3 dictionaries and langpacks aren't affected by Bug 1917613.

**Comment by eviljeff** [comment]
(testing Jira integration)
